app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -463,17 +481,13 @@ class SuggestedEditsTasksFragment : Fragment() {
     }
 
     private fun maybeShowDonorHistoryUpdatedSnackbar() {
-        if (ContributionsDashboardHelper.contributionsDashboardEnabled && ContributionsDashboardHelper.shouldShowDonorHistorySnackbar) {
-            FeedbackUtil.showMessage(this, R.string.donor_history_updated_message_snackbar)
```

Comment:
Do we still need to show the `donor history updated` message **AFTER** the user seen the survey dialog/special dialog?

For example, if the user `contributionsDashboardSurveyDialogShown` is `true`, the user will not see the "donor history updated" next time when they update the donor history.